### DISCUSSION TITLE
consult-font live preview interface similar to consult-theme

### DISCUSSION THREAD

**Discussion by ParetoOptimalDev** [minad/consult#899] [Ideas]
I just realized this would be very nice while using `menu-set-font` and thinking "what if this acted like consult-theme and let me preview the font".

**Comment by minad** [comment]
Iirc I've tried that and also a Marginalia font annotator. It was just too slow to be practical.

**Reply by ParetoOptimalDev** [reply]
For me it would be very useful if it were even limited to a small buffer, specific word or line. Not sure if that would be fast enough or useful enough for others.

**Reply by minad** [reply]
> For me it would be very useful if it were even limited to a small buffer, specific word or line. Not sure if that would be fast enough or useful enough for others.

This is what I tried in Marginalia with a font annotator. Feel free to experiment and provide a patch if you like.
